apps/portals/package.json

```diff
@@ -11,6 +11,7 @@
     "@react-hookz/web": "^23.1.0",
     "@sage-bionetworks/react-base-table": "^1.13.4",
     "@sage-bionetworks/synapse-types": "workspace:*",
+    "@tanstack/react-query": "5.22.2",
```

Comment:
Just relocated automatically by pnpm to be in lexicographical order--not changed